### PR TITLE
chore(eslint): remove unused expressions

### PR DESCRIPTION
**Pull request by DanielRuf**
## Description

This PR removes a `Object.keys()` which is not used at all.

### CODE REVIEW

**Comment by wardpeet** [comment]
@DanielRuf are you using the eslint `no-unused-expressions` rule to find this kind of issue? Or how did you spot this one? We are curious because we might want to update our eslint setup to catch this in CI.

**Comment by DanielRuf** [comment]
> @DanielRuf are you using the eslint `no-unused-expressions` rule to find this kind of issue? Or how did you spot this one? We are curious because we might want to update our eslint setup to catch this in CI.

We are currently testing and evaluating various linting tools. One of them is sonar-scanner and SonarQube / SonarCloud.

I've done an analysis at https://sonarcloud.io/dashboard?id=DanielRuf_gatsby which is the basis of my PRs.

We are currently planning to make a default setup for our developers (.vscode/extensions.json and linting configs) to help shipping better software and have additional review processes.

**Comment by wardpeet** [comment]
do you mind enabling the rule I mentioned in our eslint rule?

**Comment by DanielRuf** [comment]
> do you mind enabling the rule I mentioned in our eslint rule?

Should I do this in this PR or in another one?

**Comment by wardpeet** [comment]
In this one if you can because more things will popup and we can fix it at once.

**Comment by DanielRuf** [comment]
> In this one if you can

Done.

File: packages/gatsby/src/utils/websocket-manager.js [hunk 1]
```diff
@@ -103,8 +103,8 @@ class WebsocketManager {
     this.pageResults = new Map()
     this.staticQueryResults = new Map()
     this.errors = new Map()
-    this.websocket // eslint-disable-line no-unused-expressions
-    this.programDir // eslint-disable-line no-unused-expressions
+    // this.websocket
```
Comment (DanielRuf) [inline packages/gatsby/src/utils/websocket-manager.js]: Not sure if this is the best solution. The variable would be undefined on this context.

File: www/src/components/plugin-searchbar-body.js [hunk 2]
```diff
@@ -30,7 +30,6 @@ const searchMetaHeight = rhythm(8 / 4)
 const searchInputWrapperMargin = rhythm(3 / 4)
 
 /* stylelint-disable */
-// eslint-disable-next-line no-unused-expressions
 injectGlobal`
```
Comment (DanielRuf) [inline www/src/components/plugin-searchbar-body.js]: This throws an error when we run the `lint` task locally.

File: www/src/components/search-form.js [hunk 3]
```diff
@@ -18,7 +18,6 @@ const { curveDefault, speedDefault } = presets.animation
 const searchDropdownOffsetTop = rhythm(2)
 const homepageSearchDropdownOffsetTop = rhythm(4.5)
 
-// eslint-disable-next-line no-unused-expressions
 injectGlobal`
```
Comment (DanielRuf) [inline www/src/components/search-form.js]: Same here. Can you clarify why it was removed here?

File: packages/gatsby-source-shopify/src/lib.js [hunk 4]
```diff
@@ -46,10 +46,10 @@ export const queryAll = async (
 
   const edges = get([...path, `edges`], data)
   const nodes = edges.map(edge => edge.node)
-  // eslint-disable-next-line no-unused-expressions
-  aggregatedResponse
-    ? (aggregatedResponse = aggregatedResponse.concat(nodes))
-    : (aggregatedResponse = nodes)
+
+  aggregatedResponse = aggregatedResponse
```
Comment (DanielRuf) [inline packages/gatsby-source-shopify/src/lib.js]: This is different to the current code style in the other files.

**Pull request merged**